src/Propel/Generator/Builder/Om/AbstractOMBuilder.php

```diff
@@ -913,7 +913,7 @@ protected static function getRelatedBySuffix(ForeignKey $fk): string
             $localColumnName = $localColumn->getPhpName();
             $localTable = $fk->getTable();
             if (!$localColumnName) {
-                throw new RuntimeException(sprintf('Could not fetch column: %s in table %s.', $localColumnName, $localTable->getName()));
+                throw new RuntimeException(sprintf('Could not fetch column of table %s.', $localTable->getName()));
             }
```

Comment:
Then this was on me, since I saw sth weird/impossible going on before that:
https://github.com/propelorm/Propel2/commit/5531e13fa46f9a0ae31e0594b0a859c97ec747eb#diff-9f9a6879f541dc6ac7340c74a31dd8464e45c8d254a89930718f1080c52e65caL915

Reply:
but you were right